app/views/foreman_ansible/job_templates/capsule_update_-_ansible_default.erb

```diff
@@ -0,0 +1,79 @@
+<%#
+name: Capsule Update Playbook
+snippet: false
+template_inputs:
+- name: target_version
+  required: false
+  input_type: user
+  advanced: false
+  value_type: plain
+  hidden_value: false
```

Comment:
Where is this input used?

Reply:
I don't understand the question. I just copy/pasted the upgrade playbook and updated it to the best of my knowledge.

Reply:
The old template took `target_version` as an input and then passed it as a parameter to the installer. Now this input was dropped from the upgrade template (and its use from the corresponding installer invocation), but it is still kept here without, as far as I can see, being used anywhere. Either it should be passed to the installer or dropped from here.

Reply:
:facepalm: Now I get it! I have deleted it.